docs/get-started.md

```diff
@@ -107,160 +107,143 @@ Welcome to Nebula Graph (Version 0.1)
 nebula>
 ```
 
-Before query the dataset, you should switch to an existing graph space.
 
-```
-nebula> use nba
-Execution succeeded (Time spent: 154/793 us)
-```
-
----
+### Step 3: Build Your Own Graph
+This section describes how to build a graph and make queries. The example is built on the graph below.
 
-### Step 3 Simple Query Demo
+![Untitled Diagram (1)](https://user-images.githubusercontent.com/51590253/60649144-0774c980-9e74-11e9-86d6-bad1653e70ba.png)
 
-This query comes from a vertex walk up an edge.
-
-```
-nebula> GO FROM 5209979940224249985 OVER like
-========================
-|                   id |
-========================
-| -7187791973189815797 |
-------------------------
-|  3778194419743477824 |
-------------------------
-| -6952676908621237908 |
-------------------------
-........................
+There are three kinds of tags(course, building and team) and two edge types (select and like). The graph schema is:
+```json
+{
+     "tags": {
+       "course": ["name: string", "credits: integer"],
+       "building": ["name: string"],
+       "student": ["name: string", "age: integer", "gender: string"]
+     },
+     "edges": {
+        "select": ["grade: integer"],
+        "like": ["likeness: double"]
+     }
+}
 ```
 
-This query comes from a vertex walk up an edge and the target's age should be greater than or equal to 30, also renames the columns as `Player`, `Friend` and `Age`.
+#### Create Space
+<em>Space</em> is a region that provides physically isolation of graphs in Nebula. First we need to create a space and select it before other operations.
 
+To list all existing spaces:
+```shell
+nebula> SHOW SPACES;
 ```
-nebula> GO FROM 5209979940224249985 OVER like WHERE $$[player].age >= 30 YIELD $^[player].name AS Player, $$[player].name AS Friend, $$[player].age AS Age
-=============================================
-|          Player |            Friend | Age |
-=============================================
-| Dejounte Murray |      Kevin Durant |  30 |
----------------------------------------------
-| Dejounte Murray |        Chris Paul |  33 |
----------------------------------------------
-| Dejounte Murray |      LeBron James |  34 |
----------------------------------------------
-.............................................
-```
-
-This query comes from a vertex walk up an edge and the target's age should be greater than or equal to 30.
-
-Setting output result as input, query `Player Name`, `FromYear`, `ToYear` and `Team Name` with input's `id`.
-
-In this query, `$^` is delegated the source vertex and `$$` is the target vertex.
-
-`$-` is used to indicate the input from pipeline.
 
-(reference to `regular expressions`, using the special ^ (hat) and $ (dollar sign) metacharacters to describe the start and the end of the line.)
+To create a new space called <em>myspace_test2</em>
+```shell
+nebula> CREATE space myspace_test2(partition_num=1, replica_factor=1);
```

Comment:
CREATE SPACE. by default I prefer to use UPPER case
please check partition_num ,replica_factor UPPER or LOWER case